test/integration/minishift.go

```diff
@@ -56,6 +56,28 @@ func (m *Minishift) shouldHaveState(expected string) error {
 	return nil
 }
 
+func (m *Minishift) shouldHaveProfileState(profile string, expected string) error {
+	actual := m.runner.GetProfileStatus(profile)
+	if !strings.Contains(actual, expected) {
+		return fmt.Errorf("Profile %s of Minishift state did not match. Expected: %s, Actual: %s", profile, expected, actual)
+	}
+
+	return nil
+}
+
+func (m *Minishift) isAnActiveProfile(profileName string) error {
```

Comment:
"isAnActiveProfile" -> "isTheActiveProfile"

Reply:
Ahh..I missed it in last review comment. Will update it